src/codegen/cfg.rs

```diff
@@ -176,6 +179,34 @@ pub enum Instr {
     },
     /// Do nothing
     Nop,
+    /// Return AbiEncoded data via an environment system call
+    ReturnData {
+        data: Expression,
+        data_len: Expression,
+    },
+    /// Return a code at the end of a function
+    ReturnCode { code: ReturnCode },
```

Comment:
Just a hint / minor nitpick: For variants with single values, e.g. `ReturnCode(ReturnCode)` would make for easier syntax. But it's not a must for me in this case.

Reply:
Will fix that

Reply:
Actually, @xermicus I'm not going to change this. If you have a look at all the other elements of `Instr`, they all have named parameters even if the variant has only one attribute. I'm trying to maintain consistency.

Reply:
Sure, I favor consistency over micro optimizations any time